File: tests/openmp_examples/program_control/metadirective.4.f90
!$omp    metadirective when(user={condition(use_gpu)}: target teams distribute parallel do private(b) map(from:a(1:n))) otherwise(paralleldo)
!$omp    begin metadirective when(user={condition(run_parallel)}: parallel)
!$omp     metadirective when(construct={parallel}, user={condition(unbalanced)}: do schedule(guided) private(b)) when(construct={parallel}: do schedule(static))
!$omp    end metadirective

File: tests/openmp_examples/program_control/metadirective.4.c
#pragma omp begin declare variant match(device={kind(host)})
#pragma omp end declare variant
#pragma omp begin declare variant match(device={kind(nohost)})
#pragma omp end declare variant
#pragma omp metadirective when( user={condition(use_gpu)}: target teams distribute parallel for private(b) map(from:a[0:n]) ) otherwise( parallel for )
#pragma omp metadirective when(user={condition(run_parallel)}: parallel)
#pragma omp metadirective when( construct={parallel}, user={condition(unbalanced)}: for schedule(guided) private(b)) when( construct={parallel} : for schedule(static))
